CHANGELOG.md

```diff
@@ -12,7 +12,7 @@
 
 ## 1.0.9
 
-* pyenv: Migrate project site from https://github.com/yyuu/pyenv to https://github.com/pyenv/pyenv
+* pyenv: Migrate project site from https://github.com/pyenv/pyenv to https://github.com/pyenv/pyenv
```

Comment:
The change on this specific line doesn't make sense — it needs to be reverted. Read what it says, and you'll see why.